### ISSUE TITLE
improve sciencefair slogan, project description

### ISSUE THREAD

**Issue by aschrijver** [codeforscience/sciencefair#153]
The slogan for sciencefair is completely non-descriptive. If one only sees:

> Sciencefair -  The open source p2p desktop science library that puts users in control. 

What does it do? What is a _desktop science library_? How am I not in control now? What control?

Also the 'Why Sciencefair?' paragraph that follows does not mention how I get control back.

**Comment by aschrijver** [comment]
PS I am just saying, as I'm creating a new dat-awesome page and was adding you..

https://github.com/aschrijver/awesome-dat/blob/fresh/awesome/readme.md

**Comment by aschrijver** [comment]
PS2 A '_open source p2p desktop science library_' is probably not why your users will install and like sciencefair, that's all I'm saying :)

**Comment by aschrijver** [comment]
How about:

> Liberating and safeguarding scientific literature for the benefit of all of humanity

**Comment by jcahill** [comment]
> PS2 A 'open source p2p desktop science library' is probably not why your users will install and like sciencefair, that's all I'm saying :)

+1 on tighter language etc⁽¹⁾  
-1 on directionality.

Rationale:

1. It's always safest to assume that a broad userbase will be violently indifferent to lofty FLOSS goals.

Following the path of least resistance usually leads you to a proprietary product with a freemium business model. Having academics as your prospective users doesn't really get you very much further, apart from a small % of power users.

For a pertinent example, just look at how many among the *needs-a-reference-manager* cohort both ⒜ give lip service to Sci-Hub and ⒝ don't think twice about using & promoting Mendeley⁽²⁾ — because it serves their immediate needs with lots of happy-shiny, low-cognitive-load UX.

2. ScienceFair is positioned as a gateway drug into dat, not vice versa. Right?

So you can expect most of its prospective users to have little / no prior familiarity with dat, and to be seeking passive participation in the network — in other words, to be looking more for a ReadCube or Mendeley substitute, less for a P2P sci-authoring workflow.

As it stands, I'm not sure that the description can be substantially improved. The roadmap would need more explicit inclusion and exclusion of feature matrix-y stuff⁽³⁾ to be able to nail down something more descriptive than 'library', or maybe 'reference library'.

---
¹ ex: pictures on the main site :^)  
² owned by Elsevier  
³ ex: [Comparison of reference management software](https://en.wikipedia.org/wiki/Comparison_of_reference_management_software)

**Comment by blahah** [comment]
Thanks both of you for the suggestions and thoughtful discussion. I agree with @jcahill - I certainly think we can improve the wording, but to deviate much from the first-pass wording (in the directions suggested, or any i can think of) right now would likely not reflect the goals of the project articulately.

Right now I'm working flat out on documenting and articulating the project at a higher level. Once I've got that to a useful stage, I'll link back here and we can fruitfully attack this :)

**Comment by aschrijver** [comment]
Hi @jcahill 

> Having academics as your prospective users doesn't really get you very much further, apart from a small % of power users.

I've made a more-or-less similar argument on Dat technology as a whole. Despite the enormous enthusiasm and incredible hard work of the team and community members, I have questions if current development approach is viable in the long run. It would be very sad if it is not. I love the Dat technology ideas.

It is my impression that there may be some '[_developer myopia_](http://www.guild.net/~schwern/talks/What_Works/What_Works/slide052.html)' (similar to [marketing myopia]()) at play:

>  _When you're deep in a project it's hard to relate to people seeing it for the first time._ 

> _The Myopic cultures, Levitt postulated, would pave the way for a business to fail, due to the short-sighted mindset and illusion that a firm is in a so-called 'growth industry'. This belief leads to complacency and a loss of sight of what customers want._

For this reason for Dat I have created a lengthy discussion topic: [Positioning, vision and future direction of the Dat Project](https://github.com/datproject/discussions/issues/58)

Another thing I observed in the general development approach: [_programmer anarchy_](https://martinjeeblog.com/2012/11/20/what-is-programmer-anarchy-and-does-it-have-a-future/)

> [So] Programmer Anarchy is…
>
> - At the start of the day the programmers choose their own work during daily stand-up meetings
> - There are no PMs, Iteration Managers, BAs, QAs / testers or “managers of programmers” – all the normal rules of managing software development in a professional environment are gone. This is on the basis that formality and rules are constraining to creativity and productivity
> - It runs on the concept that with no managers to give power to their programmers to go ahead and develop (managers “empowering” their teams), programmers go ahead and take total responsibility for the success of each project in a form of self-organised “anarchy”
> - Integral to this is the adoption of the mindset “what if you were guaranteed not to fail” and the idea that disagreement and failure is expected, and both are ultimately productive outcomes. They want programmers to lose the “fear of failure”
> - Programmers work directly with the customer, which builds more trust and understanding about how the SDLC is affecting delivery
> -  And to top it off Programmer Anarchy is still Agile Manifesto compliant:
>     - Individuals and interactions over processes and tools
>     - Working software over comprehensive documentation
>     - Customer collaboration over contract negotiation
>     - Responding to change over following a plan

But not _explicitly_ recognized, chosen or followed. The anarchism follows from of the community culture of mostly scientists and/or hackers venturing in new tech field, informally structured and with ethical / moral motives that may favour an 'under-the-radar' approach.

I know these statements may be tickling. They are not meant negatively, and not as strong as might seem.

PS I am going to post this as separate issue on dat Discussions.
----

> ScienceFair is positioned as a gateway drug into dat, not vice versa. Right?

Not sure if you are right here: https://github.com/sciencefair-land/strategy (WIP)

> As it stands, I'm not sure that the description can be substantially improved. The roadmap would need more explicit inclusion and exclusion of feature matrix-y stuff⁽³⁾ 

Sure it can. Agree on the feature matrix. Existing slogan already contains some that could be listed lower down the landing page of Sciencefair ;)

**Comment by aschrijver** [comment]
@blahah 

> but to deviate much from the first-pass wording (in the directions suggested, or any i can think of) right now would likely not reflect the goals of the project accurately

I don't think this is needed. Some polishing to the landing page will already clarify much.

**Comment by blahah** [comment]
Just to comment on a few things:

> What does it do? What is a desktop science library? How am I not in control now? What control?
> Also the 'Why Sciencefair?' paragraph that follows does not mention how I get control back.

These are all excellent questions we should answer on the site when we make these changes. There are some other users who have suggested things - we should try to collect and align the suggestions.

> PS2 A 'open source p2p desktop science library' is probably not why your users will install and like sciencefair, that's all I'm saying :)

I think the ones attracted by this or something like it are probably the most useful initial users. Right now we're looking for general awareness and early technical contributors. We're far from ready for mainstream.

The same applies to:

> It's always safest to assume that a broad userbase will be violently indifferent to lofty FLOSS goals.

At some point however, and I am at least attempting to predict when this will be in the planning docs, we will want to make that switch. At that point these guidelines become relevant and crucial to the marketing of the project.

> Following the path of least resistance usually leads you to a proprietary product with a freemium business model. 

This will be guaranteed to never happen - we'll have at least a full "open lock" legal structure and hopefully a considerable pyramid of other safeguards and supports in place. You'll have to wait for details I'm afraid!

> ScienceFair is positioned as a gateway drug into dat, not vice versa. Right?

Not at all - in fact we expect users not to know or care about the technology underneath. For developers on the other hard, and a small, highly technical portion of the user base, we will be trying to help prove the benefits of decentralisation in general and of the technologies we use in particular.

> to be looking more for a ReadCube or Mendeley substitute, less for a P2P sci-authoring workflow

I doubt there's anyone in the world looking for a ReadCube substitute - everyone I've ever heard from about it is just trying to avoid it altogether. A lot of people are looking for a much better experience creating and/or consuming the literature, and over time I believe we'll deliver that in a small way and enable it in an unprecedented way. But I don't think it will ever involve chasing users of particular software (and especially not Mendeley). If we succeed in our goals, Mendeley and everything else will look like a stone tablet. If we fail, I don't want to have riled up potential enemies (more than necessary) or, worse, misled users by trying to draw them away from some other tools.

**Comment by blahah** [comment]
Also to reiterate a point I've made in the general dat threads: ScienceFair is currently not suffering from a problem of having too few people using it. Actually we've been a victim of our own success in that I've essentially had to go dark and completely restructure my life in order to keep up with the current interest, and to plan for the future.

If there comes a day when the software is achieving the basic goals of the project and not having enough users is the problem, that will be a day I stop to celebrate before returning to optimising our marketing :)

All that said, I do think we can leave users less confused and more informed, and should always be trying to do that without creating hype or disappointment. So this discussion is super valuable and we will definitely iterate the site content and other materials based on this.

**Comment by aschrijver** [comment]
Well yes, that is the point I am making all along for Dat. You should not need to go dark if there is sudden traction. But that needs preparedness.

**Comment by jcahill** [comment]
I was a little fast-and-loose with inferential gaps.

> At some point however, and I am at least attempting to predict when this will be in the planning docs, we will want to make that switch. At that point these guidelines become relevant and crucial to the marketing of the project.

Good to hear. Think that was the primary thing I was looking to voice.

> This will be guaranteed to never happen 

"Following the path of least resistance" was intended from the users' POV. As in: when most people want very little out of a tool or service, and can't readily be convinced that they should want more, mainstreaming cool FLOSS stuff has a way of becoming ridiculously hard.

[not to encourage going dark or anything, but from the project's POV it *better be guaranteed* to never happen :)]

> Not at all - in fact we expect users not to know or care about the technology underneath.

I think that's exactly what I meant to suggest by "gateway drug". Just that ScienceFair is highly visible, user-facing, and all that → if any element of the ecosystem should err on the side of user-friendliness, this'd seemingly be it.

> But I don't think it will ever involve chasing users of particular software (and especially not Mendeley). If we succeed in our goals, Mendeley and everything else will look like a stone tablet. 

sounds good.

> If we fail, I don't want to have riled up potential enemies (more than necessary)

easier for me, i just make most of my open source contributions pseudonymous!

**Comment by blahah** [comment]
Agreed, but the preparedness would be wasted for close to 100% of all projects ever. So doing it before any strong reason to believe there *will be* traction would be premature I think, especially when it starts out as one of many side projects. I hope this is the optimal time to do it for ScienceFair, and appreciate your encouragement and input :)

**Comment by blahah** [comment]
@jcahill ah, sorry yes it seems we were talking slightly across one another. It seems we're largely on the same page, and I'm sorry for reiterating what you said.

**Comment by blahah** [comment]
cc @daniellecrobinson @BettyWaitherero could you take a crack at some ideas for the tagline and description on the website? See thread above for some starting points :)

**Comment by daniellecrobinson** [comment]
Sounds good - appreciate the input everyone!

**Comment by aschrijver** [comment]
# sciencefair - fairly distributed human wisdom

**Comment by aschrijver** [comment]
btw.. the url sciencefair-app.com(mercial) is one i wouldn't use for an open-source project, especially given your mission / vision

why don't you benefit from the new domain extensions? results in attractive, easy url's
well.. an obvious one is _sciencefair.app_

cool ones:
- sciencefair.trade
- sciencefair.now
- sciencefair.protection
- sciencefair.today

others: 
- sciencefair.build
- sciencefair.channel
- sciencefair.center
- sciencefair.community
- sciencefair.direct
- sciencefair.data
- sciencefair.docs
- sciencefair.link
- sciencefair.science
- sciencefair.storage

**Comment by BettyWaitherero** [comment]
Hi Everyone!

ok my first attempt at rewriting the landing page - sorry it took me a while to literally go through everything about ScienceFair and Dat Project. I have combined with Danielle's previous paragraph as well. 

Title: "Bridging barriers to human wisdom, through decentralized file sharing networks"

Why Science Fair?
Because growing human understanding and sharing information should be easier, faster, shared, interactive, verifiable and immutable. Decentralized networks allow scientific researchers and scholars to move away from the old-world model of learning, where a library of information is situated in one central information bank. We visualize a future where peer to peer data sharing networks totally bridge barriers to scientific and scholarly knowledge; barriers such as distance, tuition fees, or accessibility which impede humanity’s intellectual progress.  Rather, with ScienceFair, growing human wisdom is inclusive and open, allowing people to quickly access research from anywhere on Earth.
ScienceFair combines decentralized file management technology, data-driven standards that support reproducibility, with a clean, modern reading interface. We allow users to curate, annotate, and share personal collections of scholarly literature. Because we believe in ceding centralized control fuels innovation, ScienceFair provides users the ability to customize their experience and control their data.

**Comment by aschrijver** [comment]
hi @BettyWaitherero 

looks good!

i would make the _Why Science Fair?_ paragraph a bit shorter still and make it 2 or 3 paragraphs

also the last sentence should be improved: "Because we believe in ceding centralized control fuels innovation, ScienceFair provides users the ability to customize their experience and control their data."

-- > because decentralization leads to innovation, we allow you to customize experience / control data

does not make sense.. one does not follow from the other :)

**Comment by BettyWaitherero** [comment]
also the last sentence should be improved: "Because we believe in ceding centralized control fuels innovation, ScienceFair provides users the ability to customize their experience and control their data."

-- > because decentralization leads to innovation, we allow you to customize experience / control data

does not make sense.. one does not follow from the other :)
^^^ please explain to me - what do you mean by, "one does not follow the other"? I am new to Dat so I am still learning how the tech works. is this a practical thing or necessarily a logic thing, that one doesn't follow the other?
ok wait - do you mean that decentralization doesn't necessarily lead to innovation? its a false conclusion/assumption, yes?

**Comment by aschrijver** [comment]
Sorry for my late response.. I am kinda busy :)

I just meant that the construction of the sentence was incorrect. Not conveying meaning well.. confusing. So:

- decentralization leads to innovation --> absolutely correct!
- so [decentralization-2-innovation] allows customize experience / control data --> unclear

I know what you mean to say. An innovation is that you get to have [..the experience of..] full control over your own data.

Rephrase that last part and you would be cool :D
